mscore/workspace.cpp

```diff
@@ -230,7 +230,8 @@ void MuseScore::changeWorkspace(Workspace* p, bool first)
 
       connect(getPaletteWorkspace(), &PaletteWorkspace::userPaletteChanged, WorkspacesManager::currentWorkspace(), QOverload<>::of(&Workspace::setDirty), Qt::UniqueConnection);
 
-      preferences.setPreference(PREF_APP_WORKSPACE, p->translatableName());
+      QString workspaceName = !p->translatableName().isEmpty() ? p->translatableName() : p->name();
```

Comment:
would it make sense to set `_translatableName` to `_name` if it is empty?

Reply:
I don't think this is a good idea because in the initialization of workspaces it is explicitly stated:
`const bool translate = isDefault || isEditedDefault;`
`if (translate)
     p->setTranslatableName(name);`

Let's not change the logic in the release. Just fix this bug

Reply:
OK